pkg/controller/garbagecollector/garbagecollector.go

```diff
@@ -60,7 +60,7 @@ const ResourceResyncTime time.Duration = 0
 // up to date as the notification is sent.
 type GarbageCollector struct {
 	restMapper    resettableRESTMapper
-	dynamicClient dynamic.DynamicInterface
+	dynamicClient dynamic.Interface
```

Comment:
we should have kept the interface name. Would have made this easier. Maybe we can restore it in a follow-up?

Reply:
:+1: now with multiple commits this is even visible :)